Packs/SkyboxSecurity/Integrations/SkyboxTicketsService/SkyboxTicketsService.py

```diff
@@ -0,0 +1,2281 @@
+import shutil
+import tempfile
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from requests import Session
+from zeep import Client as zClient
+from zeep import Settings, helpers
+from zeep.cache import SqliteCache
+from zeep.transports import Transport
+from datetime import datetime
+
+
+''' HELPER FUNCTIONS '''
+
+
+class DateTimeEncoder(json.JSONEncoder):
+    def default(self, o):
+        output = {}
+        for key in o:
+            if isinstance(o[key], datetime):
+                output[key] = o[key].isoformat()
+            else:
+                output[key] = helpers.serialize_object(o[key])
+
+        return json.dumps(output, default=lambda o: '<not serializable>')
+
+
+def serialize_object_list(input) -> List:
+    output = []
+    tmp_output = json.loads(json.dumps(input, cls=DateTimeEncoder))
+    for element in tmp_output:
+        output.append(json.loads(element))
+    return output
+
+
+def serialize_object_dict(input) -> Dict:
+    return json.loads(json.loads(json.dumps(input, cls=DateTimeEncoder)))
+
+
+def resolve_datetime(input) -> Dict:
+    output = {}
+    for key in input:
+        if isinstance(input[key], datetime):
+            output[key] = input[key].isoformat()
+        else:
+            output[key] = input[key]
+    return output
+
+
+def get_cache_path():
+    path = tempfile.gettempdir() + "/zeepcache"
+    try:
+        os.makedirs(path)
+    except OSError:
+        if os.path.isdir(path):
+            pass
+        else:
+            raise
+    db_path = os.path.join(path, "cache.db")
+    try:
+        if not os.path.isfile(db_path):
+            static_init_db = os.getenv('ZEEP_STATIC_CACHE_DB', '/zeep/static/cache.db')
+            if os.path.isfile(static_init_db):
+                demisto.debug(f'copying static init db: {static_init_db} to: {db_path}')
+                shutil.copyfile(static_init_db, db_path)
+    except Exception as ex:
+        # non fatal
+        demisto.error(f'Failed copying static init db to: {db_path}. Error: {ex}')
+    return db_path
+
+
+def getTicketWorkflow_command(client, args):
+    ticketId = args.get('ticketId')
+
+    response = client.service.getTicketWorkflow(ticketId)
+    command_results = CommandResults(
+        outputs_prefix='Skybox.getTicketWorkflow',
+        outputs=helpers.serialize_object(response),
+        raw_response=helpers.serialize_object(response)
+    )
+
+    return command_results
+
+
+def getOriginalChangeRequestRouteInfoV1_command(client, args):
+    ticketId = args.get('ticketId')
+    changeRequestId = args.get('changeRequestId')
```

Comment:
Can we stick to the current convention? For really long argument names that might be confusing to the user. 
The current convention helps as it is inline with the vendor SOAP API Documentation. The user can easily look up in the vendor doc the arguments.

For example:
```
    accessChangeTicket_customFields_lastModificationTime = args.get(
```
would be:
```
    access_change_ticket_custom_fields_last_modification_time = args.get(
```

Additionally, this will significantly extend the length of the commands in the CLI.

Reply:
Since it's partner support, we still want the arguments to be in our standard convention.

Reply:
Fixed in commits `46de42b203f8afd0cff8a552b00a98e3af35481e` and `2d7f82988ad5a4a1b1394bfc9c4d0407ae880fa9`